tests/components/habitica/test_todo.py

```diff
@@ -233,7 +217,7 @@ async def test_complete_todo_item_exception(
         (
             "todo.test_user_to_do_s",
             "88de7cd9-af2b-49ce-9afd-bf941d87336b",
-            "2024-07-30",
+            date(day=30, month=7, year=2024),
```

Comment:
I conventionally see this as `date(2024, 7, 30)` and think its similarly readable to what was there before